metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/powerbi_api.py

```diff
@@ -47,6 +57,10 @@ def __init__(self, config: PowerBiDashboardSourceConfig) -> None:
             tenant_id=self.__config.tenant_id,
         )
 
+        self.reporter = reporter
+
+        self.dataset_registry = {}
```

Comment:
needs a type annotation + add a comment explaining what the key is

Reply:
The detail is available from line number 35 to 38